admin/payment.md

```diff
@@ -10,10 +10,10 @@ From the Dashboard, locate the payment package you want to enable. Click the plu
 
 Reaction supports the following payment packages:
 
- * [PayPal](https://github.com/reactioncommerce/reaction-paypal)
- * [Braintree](https://github.com/reactioncommerce/reaction-braintree)
- * [Stripe](https://github.com/reactioncommerce/reaction-stripe)
- * [Authorize.Net](https://github.com/taylorsmithgg/reaction-auth-net)
+- [PayPal](https://github.com/reactioncommerce/reaction-paypal)
```

Comment:
Ok.  I wasn't really editing for content though.  Do you want me to update here or submit another PR?